packages/callouts/src/sass/_module.scss

```diff
@@ -0,0 +1,93 @@
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// callouts module
+// Use callouts to notify and alert users of important snippets of information.
+//
+// Content:
+// - Callout styling
+//   - Base callout placeholder selector
+//   - generic callout
+//   - --info variant
+//   - --calendar-event variant
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+
+@import '_globals.scss';
+
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Callout styling
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+/**
+ * Base callout placeholder selector.
+ */
+%base-callout {
+	margin: uikit-space( 1 ) uikit-space( 0.25 );
+	padding: uikit-space( 0.75 );
+	border-radius: 1px;
+	box-shadow: 0 0 8px $uikit-callout-box-shadow-colour;
+	max-width: $uikit-maxwidth;
+
+	.uikit-body & > p:last-child {
+		margin-bottom: 0;
+	}
+}
+
+
+/**
+ * The generic callout.
+ */
+.uikit-callout {
+	@extend %base-callout;
+}
+
+
+/**
+ * An info version of the generic callout.
+ */
+.uikit-callout--info {
+	@extend %base-callout;
```

Comment:
You don't need to extend this as all classes should have `uikit-callout` attached already. BEM